core/src/main/scala/chisel3/internal/firrtl/IR.scala

```diff
@@ -122,7 +122,6 @@ private[chisel3] object ir {
     }
 
     /** Provides a mechanism that LitArgs can have their width adjusted
-      * to match other members of a VecLiteral
```

Comment:
Stray change.

Reply:
Not stray actually, I am calling the function from Bundle literals now lol.